### ISSUE TITLE
Alchemistry blocks won't push to inventories

### ISSUE THREAD

**Issue by BrandonJ98** [al132mc/alchemistry#340]
Issue: Alchemistry machines (Dissolver, Combiner, etc) do not push to or pull from external inventories even when set to. I have tried both modded and vanilla inventories like AE2 pattern providers/interfaces as well as vanilla chests and barrels.

Minecraft version: 1.20.1
Modspack: ATM9, v0.2.56
Alchemistry version: 2.3.4 

To reproduce: 
Place an Alchemistry machine like a dissolver. Place an inventory like a chest next to it. Set the dissolver to pull or push to that chest, and place something either in the chest or machine to be processed, it refuses to pull the item in or push the item out. Item pipes still work, but Alchemistry itself does not work with pushing or pulling items.

![image](https://github.com/SmashingMods/Alchemistry/assets/42777948/48f53bf0-0e33-45d0-9cff-6f5b1cf84bd2)

**Comment by Geolykt** [comment]
In case it is relevant: Is this singleplayer or multiplayer?

**Comment by BrandonJ98** [comment]
Multiplayer, dedicated server. 

Apologies, felt like I was missing something in the post, couldn't figure out what.

**Comment by Geolykt** [comment]
Don't worry, it didn't even matter in the end (the behavior is identical in singleplayer).

After trying to search for the cause of this issue I've remembered that such functionality never existed (when I implemented that feature I deemed it to be too impactful on the performance of servers and I believe a few other mods implemented this feature similarly, but it's been a while). This behaviour might of course not be the best for UX, but compared to what it was before (being able to put items in the output slots with pipes, only being able to insert from a few specific hardcoded directions, etc.) the current behaviour is still better.

In brief the current behaviour only exists to limit what pipes can and cannot do. Implementing an Autoejection feature as already exists for the reactor multiblocks to the other blocks of Alchemistry wasn't planned by me - however I would not be opposed to this changing (that being said I myself am rather unlikely to implement it considering that I haven't played minecraft in a while and I'm the sort of person to only work on stuff I actually use).

TL;DR: WAI - sorta.
